plugins/rbac/src/hooks/useRoles.ts

```diff
@@ -1,4 +1,4 @@
-import React from 'react';
+import React, { useEffect } from 'react';
```

Comment:
@ciiay since here we are importing `React` you dont need to import `useEffect` separately.